sdk/azidentity/azidentity_test.go

```diff
@@ -27,6 +27,92 @@ const (
 	customHostString = "https://custommock.com/"
 )
 
+func getTenantDiscoveryResponse(url string) string {
+	return `{` +
```

Comment:
Backtick syntax allows multiline literals, so you don't have to concatenate every line